blog/src/Blog/Post/PostController.php

```diff
@@ -37,7 +38,7 @@ public function index(CurrentRoute $currentRoute, PostRepository $postRepository
         return $this->viewRenderer->render('index', ['item' => $item, 'canEdit' => $canEdit, 'slug' => $slug]);
     }
 
-    public function add(Request $request, ValidatorInterface $validator): Response
+    public function add(Request $request, FormHydrator $formHydrator): Response
```

Comment:
How about calling it InputHydrator? Or is it form-specific?

Reply:
It's form specific hydrator — get name from `Form` object and populate by this name.